### PR TITLE
Document scoped API keys

### PR DESCRIPTION
**Pull request by jacc**
- Add scoped API key documentation
- Calendly link

### CODE REVIEW

**Comment by stevo2105** [comment]
#### 📝 Changed routes:


| [**/software-intro**](https://new-documentation-b3p2s1hr9-whop.vercel.app/software-intro) |
|:-:|
|<a href="https://new-documentation-b3p2s1hr9-whop.vercel.app/software-intro">Screenshot of /software-intro</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/eT4fovVER2W3mP32YrIT_0jUsrxVkrWASYah0tpJHJnBw8Yvow7foc6JAtnZlu78k1ImzT_B0fjBWxcka-P3hg.png">(view full size)</a> |







Commit ecd7d5fbad33afff51fe3ede128c29394d6a224a (https://new-documentation-b3p2s1hr9-whop.vercel.app).

File: src/pages/api-keys.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+3. Set a memorable name for your key.
+4. Copy the API key
+
+After that, you're ready to start creating with Whop's API!
+
+## Scoped API keys
+
+Scoped API keys provide access to specific resources or endpoints within Whop's API, rather than providing access to the entire API. This helps to ensure that the client-side code is only able to perform the specific actions that it needs to, while also limiting the potential for security vulnerabilities.
+
+Scoped API keys are **the only way** for your company to use client-sided keys.
```
Comment (gwkline) [inline src/pages/api-keys.mdx:26]:
```suggestion
Scoped API keys are **the best way** for your company to use client-sided keys.
```

File: src/pages/api-keys.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+### Instructions to create a scoped API key
+
+1. Create a new API key under the Developers setting panel
+2. Click the triple dots and click "Edit permissions"
+3. Add the permissions you want this key to have access to
+4. Save!
+
+## Further help
+
+For more authentication help, [our developer documentation page on authentication](https://dev.whop.com/reference/authentication) can provide some more technical specifics.
```
Comment (gwkline) [inline src/pages/api-keys.mdx:39]:
```suggestion
For more authentication help, [our API reference page on authentication](https://dev.whop.com/reference/authentication) can provide some more technical specifics.
```

File: src/pages/api-keys.mdx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Scoped API keys are **the only way** for your company to use client-sided keys.
+
+![Scoped Key Setup](https://i.imgur.com/sbMzXjv.gif){{ className: 'rounded-lg w-full' }}
+
+### Instructions to create a scoped API key
+
+1. Create a new API key under the Developers setting panel
+2. Click the triple dots and click "Edit permissions"
+3. Add the permissions you want this key to have access to
```
Comment (gwkline) [inline src/pages/api-keys.mdx:34]:
```suggestion
3. Add the permissions you want this key to have access to. If you are storing keys client-side, we recommend keeping your permissions as minimal as possible. 
```

File: src/pages/api-keys.mdx [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import { Warning } from '../components/mdx'
+
+# API keys
+
+API keys are an essential tool for accessing and integrating with Whop's API. An API key is a unique identifier that allows a software product to access Whop's API and validate licenses, retrieve user information, and more.
```
Comment (gwkline) [inline src/pages/api-keys.mdx:5]:
```suggestion
API keys are an essential tool for accessing and integrating with Whop's API. An API key is a unique identifier that allows a developer to access Whop's API on behalf of a company and validate licenses, retrieve user information, and more.
```

File: src/pages/api-keys.mdx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+<Warning>
+  It's important that you keep your API keys safe and secure - unless otherwise
```
Comment (gwkline) [inline src/pages/api-keys.mdx:8]: It's important that you keep your API keys safe and secure - if not properly scoped, your API keys can be dangerous if they fall into the wrong hands.

**Review by gwkline — CHANGES_REQUESTED** [review]

**Review by gwkline — APPROVED** [review]

**Pull request merged**